tests/kafkatest/tests/streams/streams_application_upgrade_test.py

```diff
@@ -23,19 +23,18 @@
 from kafkatest.version import LATEST_2_2, LATEST_2_3, LATEST_2_4, LATEST_2_5, LATEST_2_6, LATEST_2_7, LATEST_2_8, \
   LATEST_3_0, LATEST_3_1, LATEST_3_2, LATEST_3_3, LATEST_3_4, LATEST_3_5, LATEST_3_6, LATEST_3_7, LATEST_3_8, LATEST_3_9, DEV_VERSION, KafkaVersion
 
-smoke_test_versions = [str(LATEST_2_2), str(LATEST_2_3), str(LATEST_2_4),
-                       str(LATEST_2_5), str(LATEST_2_6), str(LATEST_2_7),
-                       str(LATEST_2_8), str(LATEST_3_0), str(LATEST_3_1),
-                       str(LATEST_3_2), str(LATEST_3_3), str(LATEST_3_4),
-                       str(LATEST_3_5), str(LATEST_3_6), str(LATEST_3_7),
-                       str(LATEST_3_8), str(LATEST_3_9)]
+
+smoke_test_versions = [str(LATEST_2_4), str(LATEST_2_5), str(LATEST_2_6),
```

Comment:
@mjsax there is also a `streams_upgrade_test` which does something else but it's been a long time since I looked at the system tests, do you know/remember what the point of these different upgrade tests is?

Reply:
`streams_upgrade_test` is about the protocol. It's has a very simple "dummy" topology (`StreamsUpgradeTest.java`), and it's about version probing.

`streams_application_upgrade_test` test DSL compatibly (ie, `upgrade.from` stuff)  and uses `StreamsSmokeTest.java`.